appeals/web/src/server/appeals/appeal-details/appeal-details.router.js

```diff
@@ -42,7 +43,8 @@ router
 			permissionNames.viewAssignedCaseDetails
 		),
 		asyncHandler(controller.viewAppealDetails)
-	);
+	)
+	.post(asyncHandler(postCaseNote));
```

Comment:
Would it be worth renaming the post handler to be more generic, eg. `postAppealDetails`? Might be a case of YAGNI.